### PR TITLE
Add all states' voter reg methods handler

### PR DESCRIPTION
**Pull request by cap10morgan**
[Pivotal card](https://www.pivotaltracker.com/story/show/153681289)

### CODE REVIEW

File: src/voter_registration_works/election_authorities.clj [hunk 1]
```diff
@@ -5,12 +5,26 @@
...
+;; to the nature of the authority search (i.e. it could be for state, county,
+;; municipal, etc. authorities). BUT! Since this is voter-registration-works
+;; and states run the show pretty much everywhere when it comes to voter
+;; registration rules, are these requests typically / often / always for state
+;; authorities? If so, the "every state authority" cache could be combined with
+;; this search cache. Or at the very least you could peephole state level
+;; searches to also check the all states cache, and if still warm, pull the
+;; state you want out of that. Anyway, I didn't do any of this. - WSM 2018-7-5
+
+(def cache-ttl (* 5 60 1000))
```
Comment (theJohnnyBrown) [inline src/voter_registration_works/election_authorities.clj]: This should be in config.edn

File: src/voter_registration_works/util.clj [hunk 2]
```diff
@@ -1,8 +1,20 @@
...
+  (:require [clojure.string :as str]))
 
 (defn ->lower-case-keyword
   [in]
   (-> in
       name
       clojure.string/lower-case
       keyword))
+
+(defn ocd-id->leaf-keyword
```
Comment (theJohnnyBrown) [inline src/voter_registration_works/util.clj]: This fits better in states.clj, IMO

Comment (cap10morgan) [reply src/voter_registration_works/util.clj]: Why is that? It's not state-specific. It works with any OCD-ID.

File: src/voter_registration_works/election_authorities.clj (hunk 1 above)
Comment (cap10morgan) [reply src/voter_registration_works/election_authorities.clj]: Agreed; I'll move it.

File: src/voter_registration_works/util.clj (hunk 2 above)
Comment (theJohnnyBrown) [reply src/voter_registration_works/util.clj]: That's fair. I just saw that it's not used anywhere else and moving it there would simplify the code by removing its own `:require` and also the need for `clojure.string` in utils.

It's an aesthetic preference.

Comment (cap10morgan) [reply src/voter_registration_works/util.clj]: Yeah, that's a good argument. I've put plenty of more-generic-in-theory code in the same namespace as the only code that uses it (so far). In fact, you've convinced me. I'm moving it. :)

**Pull request merged**